<launch>
	
	<!-- Se publica la imagen de la camara -->
	<!-- <node name="image_msg_publisher" pkg="distance_meter" type="image_msg_publisher.py" output="screen"/> -->
	<!-- Launch camera -->
	<node name="usb_cam" pkg="usb_cam" type="usb_cam_node" >
		<param name="video_device" value="/dev/video4" />
		<param name="image_width" value="640" />
		<param name="image_height" value="480" />
		<param name="pixel_format" value="yuyv" />
		<param name="camera_frame_id" value="usb_cam" />
		<param name="io_method" value="mmap"/>
	</node>

	<!-- Se calcula la distancia y se la publica -->
	<node name="distance_node" pkg="distance_meter" type="distance.py"/>
	<!-- En lugar de rviz, un codigo se suscribe al topico del resultado y lo muestra -->
	<node name="image_msg_subscriber" pkg="distance_meter" type="image_msg_subscriber.py"/>
	
</launch>